src/pairwise.jl

```diff
@@ -0,0 +1,88 @@
+_allocmatrix(X, Y, T) = Matrix{T}(undef, length(X), length(Y))
+_allocmatrix(X, T) = Matrix{T}(undef, length(X), length(X))
+
+import Distances: pairwise
```

Comment:
Can you remove `import Distances: pairwise`, and just do `function Distances.pairwise()` when defining the function?

Reply:
Ok, yes will do. For my own understanding, is there a benefit to doing it this way?

Reply:
Hmm, not sure the doc string is registered in the correct way after I changed this. Please check and see if I got it wrong...

Reply:
Should be fixed now.

Reply:
No benefit — it's just how I do it in the rest of the package. I personally find it clearer.